packages/kbn-eslint-plugin-eslint/rules/no_deprecated_authz_config.test.js

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+const { RuleTester } = require('eslint');
+const rule = require('./no_deprecated_authz_config');
+const dedent = require('dedent');
+
+// Indentation is a big problem in the test cases, dedent library does not work as expected.
```

Comment:
@jbudz Maybe you can advise here on the indentation fix, is there any workaround except `dedent` (it doesn't solve nested object indentation issues)?
I've just manually adjusted indentation in most test case, like [here](https://www.reddit.com/r/meirl/comments/v6ndl8/meirl/)

Reply:
I'm not aware of anything, not seeing alternatives in the documentation or any code lookups.  Not sure.